### DISCUSSION TITLE
Can bleak return un-parsed packet

### DISCUSSION THREAD

**Discussion by EdwinFairchild** [hbldh/bleak#1355] [Q&A]
I was wondering if it would be possible to make something like a sniffer even it is only one channel at a time, but could bleak theoretically, even if its not implemented right now, give me the raw advertising packets and even connection related packets so that i can display in a graphical manner all the sections of a packet.
Like I said I know it may not be implemented but could it be, so I could take a stab at it?

**Comment by dlech** [comment]
No, Bleak doesn't have access to any low-level Bluetooth radio stuff. 

What you are describing sounds like what Wireshark already does.
